hw22(3)

```diff
@@ -0,0 +1,8 @@
+SELECT users.id, users.first_name, users.last_name, books.title
+FROM users
+LEFT JOIN purchases 
+ON users.id = purchases.user_id
+LEFT JOIN books
+ON purchases.book_id = books.id
+ORDER BY user_id;
```

Comment:
`user_id` here is a value from `purchases` which is not taken to the result. Use `users.id` here instead